src/asmcnc/apps/systemTools_app/screens/xy_jig/screen_xy_jig.py

```diff
@@ -0,0 +1,566 @@
+from kivy.uix.screenmanager import Screen
+from kivy.lang import Builder
+from kivy.clock import Clock
+
+from asmcnc.comms.yeti_grbl_protocol.c_defines import *
+from asmcnc.skavaUI import popup_info
+from asmcnc.apps.systemTools_app.screens.xy_jig.popup_xy_jig import *
+
+import matplotlib
```

Comment:
Because these screens are part of normal EC, I think these package imports will need wrapping in a try-except - if matplotlib fails to install then it could cause a crash when the whole application starts to run. 

Also, there are occasionally cases where package installs don't happen successfully in the factory on the first try, so if this screen can't load bc it's missing maplotlib, we should show the operators some kind of error message.